data/raw/keybindings.json

```diff
@@ -1138,6 +1138,19 @@
     },
     {
         "type" : "keybinding",
+        "id": "QUIT",
+        "category": "NEW_CHAR_DESCRIPTION",
```

Comment:
This will only be used on the description tab. All the other tabs will use the global "QUIT" keybindings which also includes 'q', 'Q' and 'space'. You would have to add a QUIT keybinding for every category to fix that. Don't think you can add multiple categorys, would be useful though.